must-gather/must-gather-requirements.md

```diff
@@ -0,0 +1,60 @@
+# Requirements for OpenShift sandboxed containers must-gather
+
+`must-gather` for OpenShift sandboxed contains (OSC) should gather all information and logs needed for debugging OSC.
+
+### Usage
+```sh
+oc adm must-gather --image=registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel9:latest
+```
+
+The command above will create a local directory with a dump of the OpenShift sandboxed-containers state.
+Note that this command will only get data related to the sandboxed-containers part of the OpenShift cluster.
+
+### Contents of the dump
+- All namespaces (and their children objects) that belong to any sandboxed containers resources
+
+In order to get data about other parts of the cluster (not specific to sandboxed containers) you should
+run `oc adm must-gather` (without passing a custom image). Run `oc adm must-gather -h` to see more options.
+
+#### General
+- Resource definitions
+- Service logs
+
+#### OSC
+- All namespaces and their child objects that belong to any OpenShift sandboxed containers resources
+- All OpenShift sandboxed containers custom resource definitions (CRDs)
+  - sandboxed-containers/namespaces/openshift-sandboxed-containers-operator/**_*\_description_**
+
+- Kata agent logs **???????**
+- Kata runtime logs **??????**
+
+- QEMU logs
+  - part of the **crio** logs as subsystem=qemu and subsystem=qmp
```

Comment:
The `subsystem=qemu` and `subsystem=qmp` stanzas indicate runtime logs actually, i.e. the code in the shim that starts/controls/stops QEMU. The shim does capture the actual QEMU logs and re-log them with a `qemuPid=XXXX` stanza.

See https://access.redhat.com/documentation/en-us/openshift_sandboxed_containers/1.4/html-single/openshift_sandboxed_containers_user_guide/index#sandboxed-containers-view-debug-logs_troubleshooting-sandboxed-containers to see where various logs can be found on a node.